protobuf/flow/access/access.proto

```diff
@@ -82,6 +82,12 @@ service AccessAPI {
 
   // GetNetworkParameters retrieves the Flow network details
   rpc GetNetworkParameters (GetNetworkParametersRequest) returns (GetNetworkParametersResponse);
+
+  // ProtocolState
+
+  // GetLatestProtocolStateSnapshot retrieves the latest sealed protocol state snapshot.
+  // Used by Flow nodes joining the network to bootstrap a space-efficient local state.
+  rpc GetLatestProtocolStateSnapshot (GetLatestProtocolStateSnapshotRequest) returns (ProtocolStateSnapshotResponse);
```

Comment:
This API is for retrieving a state snapshot that is used for bootstrapping the node. Prior to this point, the node has no state to work with and can't tell which messages are valid and which aren't. This snapshot is a trusted input which is then used to validate all future messages under normal operation, and it is the responsibility of the node operator to arrange to obtain a snapshot that they trust to be correct so that their node operates correctly. 

This API is intended to be the "default" method for obtaining this snapshot (using a trusted Access API), but a node operator may also want to (for example) grab a snapshot from another node that they run.